From e8c87242457ebd597c4f706b3b118afb39dbb742 Mon Sep 17 00:00:00 2001
From: Chocobozzz <me@florianbigard.com>
Date: Tue, 6 Feb 2018 08:45:48 +0100
Subject: [PATCH] Revert "Link statically to libprotobuf from the spotifyblob"

This reverts commit 170c64cd8bef9df2ea88dd4f72eec641c250bcfd.
---
 CMakeLists.txt                             | 2 --
 ext/clementine-spotifyblob/CMakeLists.txt  | 1 -
 ext/libclementine-common/CMakeLists.txt    | 1 +
 ext/libclementine-remote/CMakeLists.txt    | 1 -
 ext/libclementine-tagreader/CMakeLists.txt | 1 -
 5 files changed, 1 insertion(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 820b13851..3ec57df2b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,8 +56,6 @@ find_package(PkgConfig REQUIRED)
 find_package(Protobuf REQUIRED)
 find_package(FFTW3)
 
-find_library(PROTOBUF_STATIC_LIBRARY libprotobuf.a libprotobuf)
-
 pkg_check_modules(CDIO libcdio)
 pkg_check_modules(CHROMAPRINT REQUIRED libchromaprint)
 pkg_search_module(CRYPTOPP cryptopp libcrypto++)
diff --git a/ext/clementine-spotifyblob/CMakeLists.txt b/ext/clementine-spotifyblob/CMakeLists.txt
index ca332dca1..30553bfc4 100644
--- a/ext/clementine-spotifyblob/CMakeLists.txt
+++ b/ext/clementine-spotifyblob/CMakeLists.txt
@@ -46,7 +46,6 @@ target_link_libraries(clementine-spotifyblob
   ${QT_QTNETWORK_LIBRARY}
   ${GSTREAMER_BASE_LIBRARIES}
   ${GSTREAMER_APP_LIBRARIES}
-  ${PROTOBUF_STATIC_LIBRARY}
   clementine-spotifyblob-messages
   libclementine-common
 )
diff --git a/ext/libclementine-common/CMakeLists.txt b/ext/libclementine-common/CMakeLists.txt
index 2cf31339b..56f46722c 100644
--- a/ext/libclementine-common/CMakeLists.txt
+++ b/ext/libclementine-common/CMakeLists.txt
@@ -36,6 +36,7 @@ add_library(libclementine-common STATIC
 
 target_link_libraries(libclementine-common
   ${QT_LIBRARIES}
+  ${PROTOBUF_LIBRARY}
   ${TAGLIB_LIBRARIES}
   ${CMAKE_THREAD_LIBS_INIT}
 )
diff --git a/ext/libclementine-remote/CMakeLists.txt b/ext/libclementine-remote/CMakeLists.txt
index 5f860fbcd..f6fc7685a 100644
--- a/ext/libclementine-remote/CMakeLists.txt
+++ b/ext/libclementine-remote/CMakeLists.txt
@@ -11,7 +11,6 @@ add_library(libclementine-remote STATIC
 )
 
 target_link_libraries(libclementine-remote
-  ${PROTOBUF_LIBRARY}
   libclementine-common
 )
 
diff --git a/ext/libclementine-tagreader/CMakeLists.txt b/ext/libclementine-tagreader/CMakeLists.txt
index dd75473c9..ba9e3ff31 100644
--- a/ext/libclementine-tagreader/CMakeLists.txt
+++ b/ext/libclementine-tagreader/CMakeLists.txt
@@ -38,7 +38,6 @@ add_library(libclementine-tagreader STATIC
 )
 
 target_link_libraries(libclementine-tagreader
-  ${PROTOBUF_LIBRARY}
   libclementine-common
 )
 
-- 
2.16.1

